tests/scripts/analyze_outcomes.py

```diff
@@ -214,6 +214,324 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_cipher_aead': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_cipher_aead',
+            'component_driver': 'test_psa_crypto_config_accel_cipher_aead',
+            # Ignore suites that are being accelerated
+            'ignored_suites': [
+                'aes.cbc',
+                'aes.cfb',
+                'aes.ecb',
+                'aes.ofb',
+                'aes.rest',
+                'aes.xts',
+                'aria',
+                'camellia',
+                'ccm',
+                'chacha20',
+                'chachapoly',
+                'cipher.aes',
+                'cipher.aria',
+                'cipher.camellia',
+                'cipher.ccm',
+                'cipher.chacha20',
+                'cipher.chachapoly',
+                'cipher.des',
+                'cipher.gcm',
+                'cipher.nist_kw',
+                'cipher.padding',
+                'des',
+                'gcm.aes128_de',
+                'gcm.aes128_en',
+                'gcm.aes192_de',
+                'gcm.aes192_en',
+                'gcm.aes256_de',
+                'gcm.aes256_en',
+                'gcm.camellia',
+                'gcm.misc',
+                'cmac',
+            ],
+            'ignored_tests': {
+                # Following tests depends on AES_C/DES_C
```

Comment:
The tests don't. It's the PEM module that depends on the built-in algorithms. Same with pkcs5 and pkcs12 below.

Reply:
Agreed. I changed this (and a few other similar comments) in my local copy of 8389+this: https://github.com/Mbed-TLS/mbedtls/commit/4207c655e02add507a5510a88261dbc5ea7293b0#diff-f476aaedf3af73a4f55d7527600c29840f3b7cce70fa534b988c375e34f93dfbR278

Reply:
Ok, so I will use the same sentences so that you can skip that commit ;)